### PR TITLE
Commenting

### PR DESCRIPTION
**Pull request by Sir-Kam**
Comments describes code, outlines optional and required parameter values including their types for functions. Parameter definitions (below the function descriptions) specify default values (if optional) and display a 'maxCount' specifying the maximum 'optional' or 'included' variables that can be provided in the function's 'param' values.

### CODE REVIEW

File: Diana.lua [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+-- This does something, I think.
 do
   local strformat = string.format
   function string.format(format, ...)
     local args = {...}
     local match_no = 1
     for pos, type in string.gmatch(format, '()%%.-(%a)') do
-      if type == 't' then
-        args[match_no] = tostring(args[match_no])
+      if type == 't' then        args[match_no] = tostring(args[match_no])
```
Comment (Draycia) [inline Diana.lua]: There should be a newline after the `then`

File: Diana.lua [hunk 2]
```diff
@@ -33,14 +36,19 @@ do
...
 Diana.setKey = function(key)
   apiKey = '?api_key=' .. key;
 end
 
+-- Set region of 'Diana' to $param -> region
 Diana.setDefaultRegion = function(region)
   defaultRegion = region;
 end
 
+[[-- Get champion data.
```
Comment (Draycia) [inline Diana.lua]: Block comments in Lua are `--[[ ]]--` instead of `[[-- --]]`

Comment (Sir-Kam) [reply Diana.lua]: Hmmm okay, still colored the syntax as if it still was a comment lol

**Comment by Sir-Kam** [comment]
Updated with your change and fix to commenting.

**Comment by Draycia** [comment]
Looks good now

**Pull request merged**